grunt/esbuild/build.js

```diff
@@ -43,6 +43,15 @@ const nodeConfig = {
   platform: 'node',
   entryPoints: ['src/platform/nodejs/index.ts'],
   outfile: 'build/ably-node.js',
+  /**
+   * externals are not bundled into the esbuild output, instead they are left
+   * as commonjs imports. This means that the version installed via package.json
+   * dependencies is used rather than being pinned to whatever version we built
+   * the package release with. This is especially import in the case of 'ws' where
```

Comment:
```suggestion
   * the package release with. This is especially important in the case of 'ws' where
```

Reply:
might need to fix line width after this too

Reply:
whooops, fixed it now